modules/custom/az_google_tag/az_google_tag.module

```diff
@@ -28,3 +28,24 @@ function az_google_tag_toolbar_alter(&$items) {
     ];
   }
 }
+
+/**
+ * Checks if the consent mode config setting is enabled for any Google Tags.
+ *
+ * @return bool
+ *   TRUE if consent mode is enabled for any Google Tags, FALSE otherwise.
+ */
+function az_google_tag_consent_mode_enabled() {
+  $google_tag_storage = \Drupal::entityTypeManager()->getStorage('google_tag_container');
+  $total_count = $google_tag_storage
+    ->getQuery()
+    ->accessCheck(FALSE)
+    ->count()
+    ->execute();
+  $consent_disabled_count = count(
+    // Find Google tags with a consent mode value of 0 or FALSE.
+    $google_tag_storage->loadByProperties(['advanced_settings.consent_mode' => [0, FALSE]])
+  );
+
+  return $consent_disabled_count < $total_count;
```

Comment:
```suggestion
  $google_tag_storage = \Drupal::entityTypeManager()->getStorage('google_tag_container');
  $consent_enabled_count = count(
  // Find enabled Google tags with a consent mode value of 1 or TRUE.
    $google_tag_storage->loadByProperties([
      'advanced_settings.consent_mode' => [1, TRUE],
      'status' => [1, TRUE],
    ])
  );
  return $consent_enabled_count > 0;
```
Should we consider restricting the check to only enabled containers? Could go either way on this - perhaps warning about disabled containers with consent mode is useful too.

Reply:
I'd be OK with only checking enabled containers.  Thanks.